README.md

```diff
@@ -0,0 +1,31 @@
+What is this?
+=============
+
+Frotz.activity is a simple front end to the classic interactive fiction interpreter frotz on the XO laptop.  Frotz.activity is licensed under the GNU General Public License version 3 (see the file COPYING for details). 
```

Comment:
Let's clearly differentiate between the activity and the upstream software it is a wrapper for.  I suggest changing "Frotz.activity" to "Frotz activity" in this paragraph, and from line 8 onwards when the context is a reference to the activity instead of the Frotz upstream software.

Reply:
Thanks changed!